yos/compare.go

```diff
@@ -2,13 +2,27 @@ package yos
 
 import (
 	"bytes"
+	"fmt"
 	"io"
 	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/1set/gut/ystring"
+)
+
+var (
+	// Mask for file mode bits to compare in SameDirEntries.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comment on exported var CompareFileModeMask should be of the form "CompareFileModeMask ..."](https://app.codacy.com/manual/an9an63/gut/pullRequest?prid=4802382)